packages/mobile/src/components/share-drawer/useShareToStory.tsx

```diff
@@ -291,30 +291,36 @@ export const useShareToStory = ({
         activateKeepAwake()
         dispatch(setCancel(() => cancelStory(platform)))
         toggleProgressDrawer(true, platform)
+        dispatch(setProgress(10))
 
-        // Step 1: Render and take a screenshot of the sticker:
-        // Note: We have to capture the sticker image first because it doesn't work if you get the dominant colors first (mysterious).
-        const encodedTrackId = encodeHashId(content.track.track_id)
-        const streamMp3Url = apiClient.makeUrl(
-          `/tracks/${encodedTrackId}/stream`
-        )
-        const storyVideoPath = path.join(
-          RNFS.TemporaryDirectoryPath,
-          `storyVideo-${uuid()}.mp4`
-        )
-        const audioStartOffsetConfig =
-          content.track.duration && content.track.duration >= 20
-            ? '-ss 10 '
-            : ''
-
+        // Step 1/2: Render and take a screenshot of the sticker, calculate the dominant colors of the cover art:
```

Comment:
maybe this could just be step 1 and step 2 still. this confused me for a sec

Reply:
oo ya for sure